src/vmspawn/vmspawn.c

```diff
@@ -616,22 +616,25 @@ static int run_virtual_machine(void) {
         if (r < 0)
                 return log_oom();
 
-        if (!strv_isempty(arg_parameters)) {
-                if (ARCHITECTURE_SUPPORTS_SMBIOS) {
-                        _cleanup_free_ char *kcl = strv_join(arg_parameters, " ");
-                        if (!kcl)
-                                return log_oom();
+        char default_tty[] = "console=" DEFAULT_SERIAL_TTY;
+        r = strv_insert(&arg_parameters, 0, default_tty);
```

Comment:
`strv_prepend`

Also, why not use the string literal exactly?

Reply:
String literals are const and strv_insert takes char*.
As strv_prepend takes const char* I will change it to just a literal, thanks :)

Reply:
Fixed in d127f0cc4c.